manifests/galactic.capital.investment.yaml

```
@@ -0,0 +1,13 @@
+name: galactic.capital.investment 
+authors: zuckung
+homepage: https://github.com/zuckung/endless-sky-plugins/
+license: MIT
+version: v1.1-galactic.capital.investment
+shortDescription: Implements a short mission chain that enables repeatable job board investment opportunities which result in small daily income. Available in human, quarg and hai space.
+description: >- Implements a short mission chain that enables repeatable job board investment opportunities which result in small daily income. Available in human, quarg and hai space starting with 2 million credits cash and going up to 1000 million credits. See the readme for details.
```

```suggestion
description: >-
  Implements a short mission chain that enables repeatable job board investment opportunities which result in small daily income. Available in human, quarg and hai space starting with 2 million credits cash and going up to 1000 million credits. See the readme for details.
```
yaml multiline syntax is a mess... this should work